www/js/pages/profile/profile.template.html

```diff
@@ -316,7 +316,10 @@ <h1 class="page-title">
             <div class="row">
                 <div class="col center">
                     <span class="help-block error center" ng-if="vm.notAnyChanges">{{'financial-assessment.no_change' | translate}}</span>
-                    <button class="button button-block" ng-click="vm.updateProfile()" ng-disabled="!profileForm.$valid || vm.disableUpdateButton  || (vm.isFinancial && vm.profile.tax_residence  == null) ">
+                    <button class="button button-block"
```

Comment:
@nazaninreihani But Ashkan told that profile page for virtual account is read only. let me know how can create a virtual account without country of residence. I don't think there is a method to create a user without residence country.